src/braket/ir/jaqcd/program_v1.py

```diff
@@ -157,6 +161,23 @@ class Program(BraketSchemaBase):
 
     _PROGRAM_HEADER = BraketSchemaHeader(name="braket.ir.jaqcd.program", version="1")
     braketSchemaHeader: BraketSchemaHeader = Field(default=_PROGRAM_HEADER, const=_PROGRAM_HEADER)
-    instructions: List[GateInstructions]
+    instructions: List[Any]
     results: Optional[List[Results]]
-    basis_rotation_instructions: Optional[List[GateInstructions]]
+    basis_rotation_instructions: Optional[List[Any]]
+
+    @validator("instructions", "basis_rotation_instructions", each_item=True, pre=True)
+    def validate_instructions(cls, value, field):
+        """
+        Pydantic uses the validation subsystem to create objects. We use this custom validator for
```

Comment:
Pydantic uses the validation subsystem to create objects. This custom validator has
        2 purposes:

(note to writer: eliminate "we" for AWS style)

Reply:
ditto